app/dashboard/models.py

```diff
@@ -938,7 +938,7 @@ def receive_url(self):
         if self.web3_type == 'yge':
             return self.url
 
-        pk = self.metadata['priv_key']
+        pk = self.metadata.get('priv_key')
```

Comment:
Should we default to something in this `get` or should we use `pk = self.metadata['priv_key']` wrapped in try/except on `KeyError`?